cmd/antrea-agent/agent.go

```diff
@@ -325,21 +326,18 @@ func run(o *Options) error {
 
 	// Initialize localPodInformer for NPLAgent, AntreaIPAMController,
 	// StretchedNetworkPolicyController, and secondary network controller.
-	var localPodInformer cache.SharedIndexInformer
-	if o.enableNodePortLocal || enableBridgingMode || enableMulticlusterNP || enableFlowExporter ||
-		features.DefaultFeatureGate.Enabled(features.SecondaryNetwork) ||
-		features.DefaultFeatureGate.Enabled(features.TrafficControl) {
+	localPodInformer := lazy.New[cache.SharedIndexInformer](func() cache.SharedIndexInformer {
```

Comment:
nit: is it required to specify the type argument here?

Reply:
removed the type argument, thanks.